src/components/Overview/Overview.tsx

```diff
@@ -39,11 +41,17 @@ const Overview = (props: Props) => {
       ) : (
         <div className={classNames(styles.emptyItems, isTabletAndBelow && styles.emptyItemsMobile)} data-testid="overview-empty">
           <img src={shirt} className={styles.emptyIcon} />
-          <span className={styles.title}>{t('overview.start_dressing')}</span>
-          <span>{t('overview.get_collectibles')}</span>
-          <Button inverted as="a" href={MARKETPLACE_URL} target="_blank">
-            {t('overview.go_to_marketplace')}
-          </Button>
+          <span className={styles.title}>
+            {isLoggedInProfile ? t('overview.start_dressing') : `${profile && profile?.avatars[0].name} ${t('overview.no_collectibles')}`}
```

Comment:
If the profile does not exist or does not have a name, it will result on it being undefined, which will write `undefined`. What if we use the `getAvatarName` function?
```suggestion
            {isLoggedInProfile ? t('overview.start_dressing') : `${profile && profile?.avatars[0].name} ${t('overview.no_collectibles')}`}
```